libs/aio/odc/aio/__init__.py

```diff
@@ -292,6 +294,78 @@ def default_guide(url, depth, base):
     return step
 
 
+def s3_find_uri(uri: str, skip_check: bool):
+    """Build generator from supplied S3 URI pattern
+    Arguments:
+        uri {str} -- S3 URI to create generator for
+        skip_check {bool} -- Validity check flag
+    Raises:
+        e: ValueError if parsing URI fails
+    Returns:
+        [type] -- [description]
```

Comment:
please fill those.
Let's change output to a stream of objects rather than just strings, whatever that comes out of find.